pkg/pool-gyro/test/ECLP.test.ts

```diff
@@ -0,0 +1,217 @@
+import { ethers } from 'hardhat';
+import { expect } from 'chai';
+import { deploy, deployedAt } from '@balancer-labs/v3-helpers/src/contract';
+import { sharedBeforeEach } from '@balancer-labs/v3-common/sharedBeforeEach';
+import { Router } from '@balancer-labs/v3-vault/typechain-types/contracts/Router';
+import { ERC20TestToken } from '@balancer-labs/v3-solidity-utils/typechain-types/contracts/test/ERC20TestToken';
+import { SignerWithAddress } from '@nomicfoundation/hardhat-ethers/dist/src/signer-with-address';
+import { FP_ZERO, fp } from '@balancer-labs/v3-helpers/src/numbers';
+import {
+  MAX_UINT256,
+  MAX_UINT160,
+  MAX_UINT48,
+  ZERO_BYTES32,
+  ZERO_ADDRESS,
+} from '@balancer-labs/v3-helpers/src/constants';
+import * as VaultDeployer from '@balancer-labs/v3-helpers/src/models/vault/VaultDeployer';
+import { IVaultMock } from '@balancer-labs/v3-interfaces/typechain-types';
+import TypesConverter from '@balancer-labs/v3-helpers/src/models/types/TypesConverter';
+import { buildTokenConfig } from '@balancer-labs/v3-helpers/src/models/tokens/tokenConfig';
+import { GyroECLPPool, GyroECLPPoolFactory } from '../typechain-types';
+import { actionId } from '@balancer-labs/v3-helpers/src/models/misc/actions';
+import { MONTH } from '@balancer-labs/v3-helpers/src/time';
+import * as expectEvent from '@balancer-labs/v3-helpers/src/test/expectEvent';
+import { sortAddresses } from '@balancer-labs/v3-helpers/src/models/tokens/sortingHelper';
+import { deployPermit2 } from '@balancer-labs/v3-vault/test/Permit2Deployer';
+import { IPermit2 } from '@balancer-labs/v3-vault/typechain-types/permit2/src/interfaces/IPermit2';
+import { PoolConfigStructOutput } from '@balancer-labs/v3-solidity-utils/typechain-types/@balancer-labs/v3-interfaces/contracts/vault/IVault';
+import { TokenConfigStruct } from '../typechain-types/@balancer-labs/v3-interfaces/contracts/vault/IVault';
+
+describe('ECLP', function () {
```

Comment:
```suggestion
describe('E-CLP', function () {
```

Same here and in other comments